Frontend/src/pages/Home.js

```diff
@@ -373,7 +372,9 @@ const Home = props => {
                   cursor: "pointer"
                 }}
                 onClick={() => {
-                  window.open("https://medium.com/@dsc.tiet");
+                  window.open(
+                    "https://medium.com/developer-student-clubs-tiet"
+                  );
```

Comment:
Can this be configured to open in a new tab?

Reply:
Already doing that.

Reply:
Cool